astropy/visualization/lupton_rgb.py

```diff
@@ -0,0 +1,477 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Combine 3 images to produce a properly-scaled RGB image following Lupton et al. (2004).
+
+For details, see : http://adsabs.harvard.edu/abs/2004PASP..116..133L
+
+The three images must be aligned and have the same pixel scale and size.
+
+Example usage:
```

Comment:
Suggest to remove the example here.
A better example will be in the high-level docs page.
If you think another example in the "code" file is useful e.g. for interactive help lookup (I don't), then I'd suggest to add it to the function docstring.